### ISSUE TITLE
Update link and copy on 404 page

### ISSUE THREAD

**Issue by kristine-eudey** [hackforla/website#2129]
### Overview
We need to update the 404 page so that it offers visitors more helpful links when they reach this page so that they can get back on track and find what they're looking for. 

### Action Items

- [x] Duplicate Error page (both desktop and mobile versions) in Figma and make the below listed changes:
- [x] Replace the link to "Join Us" with a link to the sitemap page.
- [x] Update the copy to reflect this change and provide some context, writing something to the effect of "if you don’t find what you’re looking for on the homepage you might find it on the sitemap, which also shows pages currently in development"

### Resources/Instructions
[Error page - Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=10975%3A0)
[Error page - Hack for LA site](https://www.hackforla.org/sdfsdfsdg)
[Sitemap page - Hack for LA site](https://www.hackforla.org/sitemap/)

**Comment by kristine-eudey** [comment]
@janieche please provide an update on this issue. Thank you!

**Comment by janieche** [comment]
1. Progress: Not started
2. Blockers: n/a
3. Availability: 2 hours this weekend and 6 hours this upcoming week
4. ETA: 9/23
5. Pictures: n/a

**Comment by ExperimentsInHonesty** [comment]
@janieche Hope you are ok.  Can you please provide update
1. Progress
2. Blockers
3. Availability
4. ETA

**Comment by ahoang94** [comment]
@kristine-eudey I noticed that 'sitemap' is presented as both one and two words around the website (e.g., the sitemap page, the footer). We should choose one option and make it consistent across the website.

**Comment by kristine-eudey** [comment]
@ahoang94 thank you for noticing the inconsistency in spelling. We decided in Sunday's team meeting to go with "Site Map" as two words. I have created #2465 to correct inconsistencies in this spelling on the site. 

Design portion of this issue is completed and has been approved, ready for dev issue. Final design can be found here: [Figma - 404 Page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17486%3A106422)
